packages/runtime/container-runtime/src/test/gcSweepReadyUsageDetection.spec.ts

```diff
@@ -182,20 +182,41 @@ describe("Garbage Collection Tests", () => {
                 mockLocalStorage[closuresMapLocalStorageKey] = JSON.stringify({ key1: { wrongSchema: "no lastCloseTime member" } });
 
                 clock.tick(10);
-                handler.usageDetectedInInteractiveClient({});
+                handler.usageDetectedInInteractiveClient("Changed", {});
                 assert.deepEqual(JSON.parse(mockLocalStorage[closuresMapLocalStorageKey] as string), { key1: { lastCloseTime: 10 } });
                 assert.equal(closeErrors.length, 1, `Expected to close the first time`);
                 assert.equal(closeErrors[0]?.errorType ?? "", sweepReadyUsageErrorType, "Expected sweepReadyUsageErrorType the first time");
 
                 clock.tick(10);
-                handler.usageDetectedInInteractiveClient({});
+                handler.usageDetectedInInteractiveClient("Changed", {});
                 assert.equal(closeErrors.length, 1, `Should NOT have closed back-to-back due to Skip Closure Period. errors:\n${closeErrors}`);
 
                 clock.tick(oneDayMs);
-                handler.usageDetectedInInteractiveClient({});
+                handler.usageDetectedInInteractiveClient("Changed", {});
                 assert.equal(closeErrors.length, 2, `Expected to close again after waiting 1 day`);
                 assert.equal(closeErrors[1]?.errorType ?? "", sweepReadyUsageErrorType, "Expected sweepReadyUsageErrorType the second time");
             });
+            describe("interactiveClientCrashOnLoad", () => {
+                beforeEach(() => {
+                    mockLocalStorage[sweepReadyUsageDetectionKey] = "blah interactiveClientCrashOnLoad blah";
+                });
+                it("setting contains 'interactiveClientCrashOnLoad' - no-op for Changed/Revived usage", () => {
+                    createHandler().usageDetectedInInteractiveClient("Changed", {});
+                    createHandler().usageDetectedInInteractiveClient("Revived", {});
+                    assert.equal(closeErrors.length, 0, "Shouldn't close if setting doesn't include 'interactiveClientClose'");
+                });
+                it("setting contains 'interactiveClientCrashOnLoad' - throws with Loaded usage just once (due to skip)", () => {
+                    assert.throws(
+                        () => createHandler().usageDetectedInInteractiveClient("Loaded", {}),
+                        (e) => e.errorType === "unreferencedObjectUsedAfterGarbageCollected",
```

Comment:
Should the error be renamed to "objectUsedAfterGarbageCollected"? The work "unreferenced" is redundant and does not add anything.

Reply:
How about more concise "garbageObjectUsedError" - since I updated the error message to "Object used after garbage collected"

Reply:
"garbageObject" sounds funny to me :D I will leave it to you to choose the name (naming is hard!). Another option is "objectUsedAfterGC".

Reply:
Haha me too